src/ts/order.ts

```diff
@@ -69,16 +69,15 @@ export type Timestamp = number | Date;
 /**
  * Order kind.
  */
-export const enum OrderKind {
-  /**
-   * A sell order.
-   */
-  SELL,
-  /**
-   * A buy order.
-   */
-  BUY,
-}
+export type OrderKind = "sell" | "buy";
+
+/**
+ * All possible order kinds.
+ */
+export const OrderKinds = {
```

Comment:
If u don't mind to be lowercase, u can use:

```
export const OrderKinds: Record<OrderKind, OrderKind> = {
  sell: "sell",
  buy: "buy",
} as const;
```

so it complains if u forget a kind  (not sth u would forget, but anyways :P )

Reply:
Cool suggestion! I like it

Reply:
I will go with @Velenir as it makes the PR require even fewer code changes which I like.